app/utilities/repository_import_parser/importer.rb

```diff
@@ -137,7 +137,7 @@ def import_batch_to_database(full_row_import_batch)
           cell_value = column.data_type.constantize.import_from_text(
             value,
             cell_value_attributes,
-            user: @user.as_json
```

Comment:
It should be serialized

Reply:
Can you plz explain why it should be serialized.

Reply:
In case we use it in delayed jobs, to preserve user settings for that specific point in time

Reply:
@user.as_json(root: true, only: :settings)